pkg/status/status.go

```diff
@@ -358,8 +359,21 @@ func (s *status) monitorClusterStatus() {
 				break
 			}
 
-			// Wait until the operator has reconciled the minimum number of syncs.
 			_, syncEvents := s.syncRatio.GetSyncs()
+
+			// no default operator sources are present, so assume we are in a good state
+			if len(defaults.GetGlobalDefinitions()) == 0 && syncEvents == 0 {
```

Comment:
`GetGlobalDefaults` returns the defaults built into the container, not the defaults present on the cluster. I don't see how this will fix the bug that occurs when `disableAllDefaultSources` is set.

Reply:
Yep, you were correct. I misunderstood what the `GetGlobalDefaults` function actually did. please take another look